ext/node/ops/tls.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+use base64::Engine;
+use deno_core::op2;
+use webpki_root_certs;
+
+#[op2]
+#[serde]
+pub fn op_get_root_certificates() -> Vec<String> {
+  let certs = webpki_root_certs::TLS_SERVER_ROOT_CERTS
+    .iter()
+    .map(|cert| {
+      let b64 = base64::engine::general_purpose::STANDARD.encode(cert);
+      let pem_lines = b64
+        .chars()
+        .collect::<Vec<char>>()
+        .chunks(72)
```

Comment:
Should this be 64? Elsewhere in Deno we follow `rfc7468` which states:
> Generators MUST wrap the base64-encoded lines so that each line consists of exactly 64 characters except for the final line, which will encode the remainder of the data (within the 64-character line boundary), and they MUST NOT emit extraneous whitespace. Parsers MAY handle other line sizes.


 https://datatracker.ietf.org/doc/html/rfc7468#section-2

Maybe use `pem_rfc7468` crate? Ref https://github.com/denoland/deno/pull/25701#discussion_r1764864078

Reply:
node uses 72, so i had to explicitly do this

Reply:
Maybe this should be documented with a comment?